docs/registration.md

```diff
@@ -0,0 +1,9 @@
+# Application Registration
+
+To register an ALTLab application with the deployment server:
+
+1. ssh into the `altlab-gw` server.
+1. cd to `opt/deploy.altlab.dev`.
+1. Generate a secret key for the app: `sudo python3 generate-secret.py {app_name}`.
+1. This will create a key file at `{app_name}.key`. Use this key as the app secret when sending a POST request to that app's endpoint. (See the [API](./API.md) notes.)
+1. You will likely need to update the code in [`app.py`](https://github.com/UAlbertaALTLab/deploy.altlab.dev/blob/531422966d99806e13900fb17305c6cf1af55530/app/app.py#L68-L79) to handle the webhook request and run the appropriate deployment script on ALTLab's server.
```

Comment:
Is there a better, more permanent link for this?

Reply:
To `app.py`? I'm actually not sure. How do you usually reference files in docs like this? It's never been clear to me what the most stable way to do this is.

Reply:
I'm not sure either. This link is _stable_ but since it's referring to something that can be updated, it will become _out-of-date_.

A thing I can see happening is moving the deployment commands to their own file (and _just_ the deployments!) and then linking to that file, and hope its name and format doesn't change 😅

Reply:
I like the idea of a separate deployments file! Could that be incorporated into #5?

At the very least, I'll update this to just point to the filename rather than specific lines. Presumably that's slightly less likely to go out of date haha. I'll point it to the deployments file if you decide to make that change in #5.

Reply:
It has been incorporated into #5! The file is `app/configuration.py`!

Reply:
Changed the docs to point to `app/configuration.py` (without a link).

Reply:
Because the target file is in the same repo, you can use a relative link in the markdown, and the GitHub UI will render a working link ``[`configuration.py`](../app/configuration.py)``